node/perf-test/src/command.rs

```diff
@@ -203,11 +203,12 @@ where
 
 		let command_sink_for_deps = Some(command_sink.clone());
 
-		let block_data_cache = Arc::new(fc_rpc::EthBlockDataCache::new(
+		let block_data_cache = Arc::new(fc_rpc::EthBlockDataCacheTask::new(
```

Comment:
@librelois is this using the new limit in size and not blocks ?

Reply:
yes

Reply:
So we should use different values no? What do you suggest @librelois ?

Reply:
The new default value is 300000000 (~300Mb)

Reply:
ok